packages/libs/react-components/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, SystemIcons } from './../../';
+import { fullWidthVariant, stackVariant } from './styles';
+import { Card, ICardProps } from '.';
+import { CardBody, CardFooter, CardHeading } from './';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<ICardProps> = {
+  title: 'Layout/Card',
+  argTypes: {
+    stack: {
+      options: Object.keys(stackVariant) as (keyof typeof stackVariant)[],
+      control: {
+        type: 'select',
+      },
+    },
+    fullWidth: {
+      options: Object.keys(
+        fullWidthVariant,
+      ) as (keyof typeof fullWidthVariant)[],
+      control: {
+        type: 'select',
+      },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<ICardProps>;
+
+export const Primary: Story = {
+  name: 'Card',
+  args: {
+    stack: false,
+    fullWidth: false,
+  },
+  render: ({ stack, fullWidth }) => {
+    return (
+      <>
+        <Card stack={stack} fullWidth={fullWidth}>
+          <CardHeading>Getting Started is Simple</CardHeading>
+          <CardBody>
+            Learn Kadena&apos;s core concepts & tools for development in 15
+            minutes
+          </CardBody>
+          <CardFooter>
+            <Button title={'Button'} icon={SystemIcons.Information}>
+              Get Started
+            </Button>
+            <Button title={'Button'}>Hello World Tutorial</Button>
+          </CardFooter>
+        </Card>
```

Comment:
For future implementations: Sub components should be namespaced so you can use them with the following syntax
```
<Card>
  <Card.Heading>
    ...
  </Card.Heading>
</Card>
```

You can see an example of this in the export for the Grid component!